index.html

```diff
@@ -216,6 +216,55 @@ <h2>Skills</h2>
                     </div>
                 </div>
             </section>
+            <!------------------------------------Contact section------------------------------------>
+            <section class="contact-form">
+                <div class="contact-header">
+                    <h1>I'm always interested in hearing about new projects, so if you'd 
+                    like to chat please get in touch</h1>
+                </div>
+                <div class="form-container">
+                    <form class="contact">
+                        <input type="text" placeholder="Full name" maxlength="30" required>
+                        <input type="email" placeholder="Email adderss" required>
+                        <textarea type="textarea" maxlength="500" placeholder="Hey I'd like ot connect to..."></textarea>
+                        <button class="btn">Get in Touch</button>
+                    </form>
```

Comment:
- [ ] As the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) suggest, kindly ensure that the form collects data by using the [Formspree service](https://formspree.io/html). This will enable you to be able to receive the form submitted in personal email through the unique id generated in formspree ☺️.

Reply:
Dear @AlphaNtihinduka,
Thanks for the comment, I did this with my codding partners